lib/mongoid/orderable/mixins/changeable.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module Mongoid
+module Orderable
```

Comment:
File needs some indent.

Reply:
In this project we aren't indenting multiple modules. Rubocop is set accordingly:

```
Layout/IndentationWidth:
  AllowedPatterns:
    - '^\s*module'
```